JS/test/airport.spec.js

```diff
@@ -66,30 +66,28 @@ describe('My Test', () => {
             }
         }
         assert.isTrue(nextPlaneMaxLoadCapacityIsHigherThanCurrent);
-    })
+    });
 
-    it('testHasAtLeastOneBomberInMilitaryPlanes', () => {
+    it('List of Bomber planes In Military Planes', () => {
         let airport = new Airport(planes);
-        let bomberMilitaryPlanes  = airport.getBomberMilitaryPlanes ();
+        let bomberMilitaryPlanes  = airport.getBomberMilitaryPlanes();
         let flag = false;
         for (let militaryPlane of bomberMilitaryPlanes) {
-            if (militaryPlane.getMilitaryType() === MilitaryType.BOMBER) {
+            if (militaryPlane.getMilitaryType() == MilitaryType.Bomber) {
                 flag = true;
             }
             else{
-                assert.fail("Test failed!");
+                assert.fail("No Bomber plains exist!");
```

Comment:
Please correct this code. There is an additional logic implemented. It can be written shorter without any `flag` variable.

Reply:
assert.isNotEmpty() was added